cfgov/unprocessed/root/data.json

```diff
@@ -142,7 +122,7 @@
 
       "spatial": "United States",
 
-      "temporal": "2011-12-01/2013-11-06",
+      "temporal": "2011-12-01/2020-01-17",
```

Comment:
The `temporal` field [is described](https://project-open-data.cio.gov/v1.1/schema/) as:

> The range of temporal applicability of a dataset (i.e., a start and end date of applicability for the data).

Given that, if we can't update the end date on a rolling basis, I would suggest we omit this field entirely.